django_reference_app/django_ref_config/settings.py

```diff
@@ -85,8 +86,10 @@
         'USER': os.getenv('POSTGRES_USER'),
         'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
         'HOST': os.getenv('POSTGRES_HOST'),
-        'PORT': os.getenv('POSTGRES_PORT', 5432)
-
+        'PORT': os.getenv('POSTGRES_PORT', 5432),
+        'OPTIONS': {
+            'connect_timeout': 5,
```

Comment:
It'd be good to add a comment here explaining why this value has been set (to ensure the app exits if the database connection times out). That should also be added to user docs as a common gotcha, but it'd be good to explain it in-place as well.

Reply:
Done under ce67b12.